sb_vision/tokens.py

```diff
@@ -93,6 +94,9 @@ def _get_distance_model(name: str, image_size: Tuple[int, int]) -> Dict[str, Any
             ),
         )
 
+    if 'marker_size' not in calibration:
+        raise ValueError("Calibrations must have a marker_size field")
```

Comment:
If we're going to require this at the usage of the distance model then we should also require it at the generation of the model -- i.e: in the calibration logic.

Reply:
Obviously if we change the requirements of the calibration annotations file we'll need to increment the version number.